### ISSUE TITLE
Orphanet evidence lacks variant functional consequences

### ISSUE THREAD

**Issue by ireneisdoomed** [opentargets/issues#2555]
**Describe the bug**
`variantFunctionalConsequenceId` is null for all Orphanet evidence, even though a fraction of the evidence reports whether the disease causing mutation results in a loss or gain of function.

**Observed behaviour**
The type of the association established in the Orphanet evidence is described in their raw data. With this, we can populate the `variantFunctionalConsequenceId` field for those which result in a loss or gain of function:
```
CONSEQUENCE_MAP = {
    "Disease-causing germline mutation(s) (loss of function) in": "SO_0002054",
    "Disease-causing germline mutation(s) in": None,
    "Modifying germline mutation in": None,
    "Disease-causing germline mutation(s) (gain of function) in": "SO_0002053"
}
```

This is already handled in the parser, but the field is not present in the final output just because `variantFunctionalConsequenceId` is missing from [the final `select` statement](https://github.com/opentargets/evidence_datasource_parsers/blob/619625e72433afad14f1c7fe986f3fece7ca84d1/modules/Orphanet.py#L174). I'm fixing this in the `22.04` branch of the evidence data parsers.

**Expected behaviour**
This info will be part of the final evidence set. This is their frequency:
```
+------------------------------+-----+
|variantFunctionalConsequenceId|count|
+------------------------------+-----+
|                    SO_0002053|  210|
|                    SO_0002054| 1175|
|                          null| 4981|
+------------------------------+-----+
```

**Questions**
Since this was not in the data, we weren't displaying it in the FE. Given the % of missingness, is it something that we want to add? @d0choa

**Comment by d0choa** [comment]
When available, this information is very relevant. I would aim to include it in the FE.

**Comment by ireneisdoomed** [comment]
Fixed in [#122](https://github.com/opentargets/evidence_datasource_parsers/pull/122)
